packages/edge-login-ui-rn/src/components/navigation/Router.js

```diff
@@ -49,7 +49,7 @@ class RouterComponent extends React.Component<Props> {
     const { ScreenStyle } = Styles
     return (
       <ThemeProvider>
-        <Airship avoidAndroidKeyboard statusBarTranslucent>
+        <Airship>
```

Comment:
These properties were part of an older version of `react-native-airship`, but it no longer understands or accepts them since we re-worked its layout algorithm.